Docs refactor

joshreini1 opened the pull request:
Merge welcome/home page, update copy. Explain still available.

Screenshots:

![Screenshot 2024-01-25 at 6 29 29 PM](https://github.com/truera/trulens/assets/60949774/fb8bf260-9579-4db3-9e18-9c06b84244f7)
![Screenshot 2024-01-31 at 4 38 30 PM](https://github.com/truera/trulens/assets/60949774/1af1927d-d715-487e-800d-257a1e546c53)

shayaks reviewed (approved):
Can we make the top bar a bit more noticable?

joshreini1 commented:
> Can we make the top bar a bit more noticable?

Could not make the top bar more noticable. Instead, moved back to left bar and removed the sections so that it all displays in reasonable space.

joshreini1 commented:
Deployed to trulens.org
